package.json

```
@@ -11,7 +11,7 @@
     "build": "npm run build-only -- --lint-spec --strict",
     "build-for-pdf": "npm run build -- --old-toc",
     "test": "npm run build-to -- --lint-spec /dev/null",
```

Might as well fix `test` while we're at it.
```suggestion
    "test": "npm run build-to -- /dev/null --lint-spec --strict",
```

---

Both versions work for me but yeah seems like a change I'm more than happy to take!